src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class AbstractFigure implements Figure {
```

Comment:
implement newly created interfaces here, you don't need to override their methods here, only in subclasses